algorithms/kernel/adaboost/adaboost_train_impl.i

```diff
@@ -98,6 +98,7 @@ services::Status AdaBoostTrainKernel<method, algorithmFPType, cpu>::adaboostSAMM
     service_memset<algorithmFPType, cpu>(w, invNVectors, nVectors);
 
     services::SharedPtr<classifier::training::Batch> learnerTrain = parameter->weakLearnerTraining->clone();
+    learnerTrain->enableChecks(false);
```

Comment:
We checked this before and don't want to check this again, right?

Reply:
yes

Reply:
yes